src/index.ts

```diff
@@ -38,7 +44,7 @@ ast.getSourceFiles().forEach(sourceFile => {
     }
 
     const sourcePath = sourceFile.getFilePath();
-    const outputPath = sourcePath.replace(rootPath + '/src', rootPath + '/test');
+    const outputPath = sourcePath.replace( srcDir, outDir );
```

Comment:
Replace the `.ts` suffix with `.mock.ts` here. At the moment, the variable value is incorrect.